modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -144,18 +144,20 @@ a patch version increase occurs.
 |Accessibility |Restrict permissions |Relax permissions
 |===
 
-=== When to Introduce a New Version
+=== When to Introduce a New Major Version
 
-You should introduce a new version for:
+You should introduce a new major version for:
 
-* Quality of service (QoS) -- When a change affects the quality of service, such as response time and availability, the API version may need to change as the QoS can break applications that use the API.
-* Output message domain value changes -- Applications that use the API depend on the domain of values of fields in response messages. For instance, the API may only return certain products or specific payment methods. If the API starts returning values outside of a domain, it can adversely affect existing consumers. In this case, consider introducing a new API version.
+* Quality of service (QoS) -- When a change affects the quality of service, such as response time and availability, the major version and API version may need to change as the QoS can break applications that use the API.
+* Output message domain value changes -- Applications that use the API depend on the domain of values of fields in response messages. For instance, the API may only return certain products or specific payment methods. If the API starts returning values outside of a domain, it can adversely affect existing consumers. In this case, consider introducing a new major version and a new API version.
```

Comment:
```suggestion
* Output message domain value changes
Applications that use the API depend on the domain of values of fields in response messages. For instance, the API might return only certain products or specific payment methods. If the API starts returning values outside of a domain, it can adversely affect existing consumers. In this case, consider introducing a new major version and a new API version.
```